libs/coin-modules/coin-ton/src/bridge/bridgeHelpers/api.ts

```diff
@@ -82,6 +86,37 @@ export async function fetchAccountInfo(addr: string): Promise<TonAccountInfo> {
   };
 }
 
+export async function fetchJettonTransactions(
+  addr: string,
+  opts?: {
+    jettonMaster?: string;
+    startLt?: string;
+    endLt?: string;
+  },
+): Promise<TonJettonTransfer[]> {
+  const address = Address.parse(addr);
+  const urlAddr = address.toString({ bounceable: false, urlSafe: true });
+  let url = `/jetton/transfers?address=${urlAddr}&limit=256`;
+  if (opts?.jettonMaster != null) url += `&jetton_master=${opts.jettonMaster}`;
+  if (opts?.startLt != null) url += `&start_lt=${opts.startLt}`;
+  if (opts?.endLt != null) url += `&end_lt=${opts.endLt}`;
+  return (await fetch<TonResponseJettonTransfer>(url)).jetton_transfers;
+}
+
+export async function fetchJettonWallets(opts?: {
+  address?: string;
+  jettonMaster?: string;
+}): Promise<TonJettonWallet[]> {
+  let url = `/jetton/wallets?limit=256`;
+  if (opts?.jettonMaster != null) url += `&jetton_address=${opts.jettonMaster}`;
+  if (opts?.address != null) {
```

Comment:
If I understand well, the address and the jettonMaster should never be null in Ledger Live. Isn't it?

Reply:
Yes, this information will always be in Ledger Live. I'll delete the conditional